polymer.json

```diff
@@ -2,9 +2,29 @@
   "entrypoint": "index.html",
   "shell": "src/lancie-admin-app.html",
   "sourceGlobs": [
-   "images-optimized/**/*"
+   "images-optimized/**/*",
+   "scripts/*"
```

Comment:
We do not have scripts yet, and may never have them. In the frontend we use them for data. I'm pretty sure we don't need that in this repo.